app/src/main/res/layout/activity_main.xml

```diff
@@ -17,6 +17,18 @@
         android:layout_width="match_parent"
         app:layout_constraintBottom_toBottomOf="parent"
         app:layout_constraintTop_toTopOf="parent" />
+
+    <com.google.android.material.floatingactionbutton.FloatingActionButton
+        android:id="@+id/fab"
+        android:layout_width="@dimen/fab_width"
+        android:layout_height="wrap_content"
```

Comment:
Pls use the same values for width and height : 60x60 or wrapxwrap

Reply:
Done thx